frontend/packages/design-system/src/themes/Input.ts

```diff
@@ -22,7 +22,7 @@ export default {
     color: 'neutral.300',
     padding: 3,
     _web: {
-      ...textTheme.variants.bodyText,
+      ...textTheme.variants.body1,
```

Comment:
Similar point to the button, but I think there's a stronger reason to keep an input in sync with body text so this may make sense.